esp32-wroom-rp/src/spi.rs

```diff
@@ -7,8 +7,8 @@ use super::protocol::{
 };
 use super::{Error, FirmwareVersion, WifiCommon, ARRAY_LENGTH_PLACEHOLDER};
 
-use eh_02::blocking::spi::Transfer;
-use embedded_hal::delay::blocking::DelayUs;
+use embedded_hal::blocking::delay::DelayMs;
+use embedded_hal::blocking::spi::Transfer;
```

Comment:
Really happy about using one version of the `embedded_hal` crate

Reply:
Me too! I'm glad we figured this one out!